File: catalogue-api/src/test/java/io/quartic/catalogue/api/GeoJsonDatasetLocatorShould.java
package io.quartic.catalogue.api;

public class GeoJsonDatasetLocatorShould extends DatasetLocatorTests<GeoJsonDatasetLocator> {
    @Override
    protected GeoJsonDatasetLocator locator() {
        return GeoJsonDatasetLocatorImpl.of("http://wat");
    }

    @Override
    protected String json() {
        return "{\"type\": \"geojson\", \"path\": \"http://wat\"}";
    }
}

File: catalogue-api/src/test/kotlin/io/quartic/catalogue/api/GeoJsonDatasetLocatorShould.kt
package io.quartic.catalogue.api

class GeoJsonDatasetLocatorShould : DatasetLocatorTests<GeoJsonDatasetLocator>() {
    override fun locator() = GeoJsonDatasetLocator("http://wat")

    override fun json() = "{\"type\": \"geojson\", \"url\": \"http://wat\"}"
}
